cdap-docs/admin-manual/source/operations/preferences.rst

```diff
@@ -76,4 +76,32 @@ or provide Twitter API credentials::
       queue = new LinkedBlockingQueue<Tweet>(10000);
       collector = new CollectingThread();
       collector.start();
-    }
\ No newline at end of file
+    }
+
+Scoped Runtime Arguments for Workflow
+=====================================
+When Workflow is configured, you may want to pass different runtime arguments to different programs
+and datasets used inside the program. You can prefix the runtime arguments with a ``<scope>.`` to achieve this.
```

Comment:
inside the Workflow. To achieve this, you can prefix the runtime arguments with a ``<scope>``.